app/controllers/declaration/DeclarantDetailsController.scala

```diff
@@ -43,14 +43,14 @@ class DeclarantDetailsController @Inject()(
 
   def displayPage(mode: Mode): Action[AnyContent] = (authenticate andThen journeyType) { implicit request =>
     request.cacheModel.parties.declarantDetails match {
-      case Some(data) => Ok(declarantDetailsPage(mode, DeclarantDetails.form().fill(data)))
-      case _          => Ok(declarantDetailsPage(mode, DeclarantDetails.form()))
+      case Some(data) => Ok(declarantDetailsPage(mode, DeclarantDetails.form(request.declarationType).fill(data)))
```

Comment:
I've created a private method in my controllers to be able to continue construct the form(). E.g.

```private def form()(implicit request: JourneyRequest[_]) = DepartureTransport.form(request.declarationType)```

Reply:
Fixed. Thanks!